packages/dd-trace/test/appsec/sdk/index.spec.js

```diff
@@ -0,0 +1,211 @@
+'use strict'
+
+const proxyquire = require('proxyquire')
+const agent = require('../../plugins/agent')
+const getPort = require('get-port')
+const axios = require('axios')
+const appsec = require('../../../src/appsec')
+const tracer = require('../../../../../index')
+const Config = require('../../../src/config')
+
+describe('Appsec SDK', () => {
+  describe('calls to external methods', () => {
+    let trackUserLoginSuccessEvent, trackUserLoginFailureEvent, trackCustomEvent
+    let appsecSdk
+    const tracer = {}
+
+    beforeEach(() => {
+      trackUserLoginSuccessEvent = sinon.stub()
+      trackUserLoginFailureEvent = sinon.stub()
+      trackCustomEvent = sinon.stub()
+
+      const AppsecSdk = proxyquire('../../../src/appsec/sdk', {
+        './track_event': { trackUserLoginSuccessEvent, trackUserLoginFailureEvent, trackCustomEvent }
+      })
+
+      appsecSdk = new AppsecSdk(tracer)
+    })
+
+    it('trackUserLoginSuccessEvent should call track_event trackUserLoginSuccessEvent', () => {
+      const user = { id: 'user_id' }
+      const metadata = { key: 'value' }
+      appsecSdk.trackUserLoginSuccessEvent(user, metadata)
+
+      expect(trackUserLoginSuccessEvent).to.have.been.calledOnceWithExactly(tracer, user, metadata)
+    })
+
+    it('trackUserLoginFailureEvent should call track_event trackUserLoginFailureEvent', () => {
+      const user = { id: 'user_id' }
+      const metadata = { key: 'value' }
+      appsecSdk.trackUserLoginFailureEvent(user, metadata)
+
+      expect(trackUserLoginFailureEvent).to.have.been.calledOnceWithExactly(tracer, user, metadata)
```

Comment:
This is just copy pasted from the test above, but `trackUserLoginFailureEvent()` have different arguments: `userId` instead of `user`, and missing `exists` argument.

I found this error in the test by using `calledOnceWithExactly` instead of `calledOnceWith`, and the test suddenly failed [here](https://github.com/DataDog/dd-trace-js/actions/runs/4032296117/jobs/6932085022), this is proof that it's useful ;)